<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DepositCalculator</class>
 <widget class="QMainWindow" name="DepositCalculator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>644</width>
    <height>501</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Times New Roman</family>
   </font>
  </property>
  <property name="cursor">
   <cursorShape>PointingHandCursor</cursorShape>
  </property>
  <property name="acceptDrops">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Депозитный калькулятор</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: gray;
 ; color: rgb(254, 255, 254); border: none;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="depositCalcButton">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>420</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
     background-color: rgb(246, 153, 6); color: rgb(245, 246, 246); border-radius: 15px;  margin: 4px;
   
    }
QPushButton:pressed {  
    background-color: rgb(245, 246, 246);
	color: rgb(246, 153, 6);
 }

</string>
    </property>
    <property name="text">
     <string>Расчитать</string>
    </property>
   </widget>
   <widget class="QPushButton" name="depositCleanButton">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>420</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
     background-color: rgb(165, 165, 165); color: rgb(0, 0, 0); border-radius: 15px;  margin: 4px;
   
    }
QPushButton:pressed {  
    background-color: rgb(0, 0, 0);
	color: rgb(165, 165, 165);
 }

</string>
    </property>
    <property name="text">
     <string>Очистить</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelHistory_2">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>10</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: gray;
 ; color: rgb(254, 255, 254); border: none;</string>
    </property>
    <property name="text">
     <string>Депозитный калькулятор</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelHistory_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: gray;
 ; color: rgb(254, 255, 254); border: none;</string>
    </property>
    <property name="text">
     <string>Сумма вклада</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelHistory_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: gray;
 ; color: rgb(254, 255, 254); border: none;</string>
    </property>
    <property name="text">
     <string>Срок (в месяцах)</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelHistory_5">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>70</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: gray;
 ; color: rgb(254, 255, 254); border: none;</string>
    </property>
    <property name="text">
     <string>Процентная ставка (%)</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="sumSpinBox">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>70</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"> background-color: rgb(254, 255, 254); color: rgb(0, 0, 0); border: none;</string>
    </property>
    <property name="decimals">
     <number>2</number>
    </property>
    <property name="minimum">
     <double>0.000000000000000</double>
    </property>
    <property name="maximum">
     <double>100000000.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>100000.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="termSpinBox">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>110</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"> background-color: rgb(254, 255, 254); color: rgb(0, 0, 0); border: none;</string>
    </property>
    <property name="decimals">
     <number>0</number>
    </property>
    <property name="minimum">
     <double>0.000000000000000</double>
    </property>
    <property name="maximum">
     <double>720.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>12.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="percentSpinBox">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>70</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"> background-color: rgb(254, 255, 254); color: rgb(0, 0, 0); border: none;</string>
    </property>
    <property name="decimals">
     <number>2</number>
    </property>
    <property name="minimum">
     <double>0.000000000000000</double>
    </property>
    <property name="maximum">
     <double>100.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="labelHistory_6">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>330</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: gray;
 ; color: rgb(254, 255, 254); border: none;</string>
    </property>
    <property name="text">
     <string>Начисленные проценты</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelHistory_7">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>370</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: gray;
 ; color: rgb(254, 255, 254); border: none;</string>
    </property>
    <property name="text">
     <string>Сумма налога</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelHistory_8">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>290</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: gray;
 ; color: rgb(254, 255, 254); border: none;</string>
    </property>
    <property name="text">
     <string>Итоговая сумма на вкладе</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="totalAmountLineEdit">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>290</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::DefaultContextMenu</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true"> background-color: rgb(254, 255, 254); color: rgb(0, 0, 0); border: none;</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="maxLength">
     <number>255</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <property name="dragEnabled">
     <bool>false</bool>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="cursorMoveStyle">
     <enum>Qt::LogicalMoveStyle</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="totalPercenttLineEdit">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>330</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::DefaultContextMenu</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true"> background-color: rgb(254, 255, 254); color: rgb(0, 0, 0); border: none;</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="maxLength">
     <number>255</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <property name="dragEnabled">
     <bool>false</bool>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="cursorMoveStyle">
     <enum>Qt::LogicalMoveStyle</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="totalTaxLineEdit">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>370</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::DefaultContextMenu</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true"> background-color: rgb(254, 255, 254); color: rgb(0, 0, 0); border: none;</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="maxLength">
     <number>255</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <property name="dragEnabled">
     <bool>false</bool>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="cursorMoveStyle">
     <enum>Qt::LogicalMoveStyle</enum>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="percentTaxSpinBox">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>110</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"> background-color: rgb(254, 255, 254); color: rgb(0, 0, 0); border: none;</string>
    </property>
    <property name="decimals">
     <number>2</number>
    </property>
    <property name="minimum">
     <double>0.000000000000000</double>
    </property>
    <property name="maximum">
     <double>100.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="labelHistory_9">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>110</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: gray;
 ; color: rgb(254, 255, 254); border: none;</string>
    </property>
    <property name="text">
     <string>Налоговая ставка (%)</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QListWidget" name="frequencyListWidget">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>160</y>
      <width>151</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"> background-color: rgb(254, 255, 254); color: rgb(0, 0, 0); border: none;</string>
    </property>
    <property name="movement">
     <enum>QListView::Free</enum>
    </property>
    <property name="currentRow">
     <number>2</number>
    </property>
    <item>
     <property name="text">
      <string>Каждый день</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Каждую неделю</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Раз в месяц</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Раз в квартал</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Раз в полгода</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Раз в год</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>В конце срока</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="labelHistory_10">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>160</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: gray;
 ; color: rgb(254, 255, 254); border: none;</string>
    </property>
    <property name="text">
     <string>Периодичность выплат</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QCheckBox" name="capitalizationCheckBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: gray;
 ; color: rgb(254, 255, 254); border: none;</string>
    </property>
    <property name="text">
     <string>Капитализация процентов</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
    <property name="tristate">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QTableWidget" name="depositsTableWidget">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>-460</y>
      <width>601</width>
      <height>171</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QTableWidget::section{background-color: green; color: black}
QTableWidget{background-color: white; color: black}
</string>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::MultiSelection</enum>
    </property>
    <property name="sortingEnabled">
     <bool>false</bool>
    </property>
    <attribute name="horizontalHeaderCascadingSectionResizes">
     <bool>true</bool>
    </attribute>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>179</number>
    </attribute>
    <column>
     <property name="text">
      <string>Месяц</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Сумма пополнения</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Сумма снятия</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="addEntryButton">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>-390</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
     background-color: rgb(246, 153, 6); color: rgb(245, 246, 246); border-radius: 15px;  margin: 4px;
   
    }
QPushButton:pressed {  
    background-color: rgb(245, 246, 246);
	color: rgb(246, 153, 6);
 }

</string>
    </property>
    <property name="text">
     <string>Добавить </string>
    </property>
   </widget>
   <widget class="QPushButton" name="depositTableClenaButton">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>-450</y>
      <width>251</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
     background-color: rgb(165, 165, 165); color: rgb(0, 0, 0); border-radius: 15px;  margin: 4px;
   
    }
QPushButton:pressed {  
    background-color: rgb(0, 0, 0);
	color: rgb(165, 165, 165);
 }

</string>
    </property>
    <property name="text">
     <string>Очистить таблицу</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelHistory_11">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: gray;
 ; color: rgb(254, 255, 254); border: none;</string>
    </property>
    <property name="text">
     <string>Сумма без налога</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="amoutWithoutTaxSpinBox">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>160</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"> background-color: rgb(254, 255, 254); color: rgb(0, 0, 0); border: none;</string>
    </property>
    <property name="decimals">
     <number>2</number>
    </property>
    <property name="minimum">
     <double>0.000000000000000</double>
    </property>
    <property name="maximum">
     <double>100000000.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>100000.000000000000000</double>
    </property>
    <property name="value">
     <double>50000.000000000000000</double>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
